packages/language-core/src/types.ts

```
@@ -108,6 +108,19 @@ export type MpxLanguagePluginReturn = {
     sfc: Sfc,
     embeddedFile: MpxEmbeddedCode,
   ): void
+  resolveUsingComponentsPath?(
+    usingComponentsPath: Map<string, UsingComponentInfo>,
+    uri: string,
+  ): Promise<Map<string, ResolvedUsingComponentInfo> | void> | void
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)